source/content/modules-known-issues.md

```diff
@@ -291,6 +291,20 @@ $conf[‘schema_suppress_type_warnings’] = TRUE;
 
 ___
 
+## [Search Api Solr Date Sort](https://www.drupal.org/project/search_api_solr_date_sort)
+
+<ReviewDate date="2020-02-14" />
+
+**Issue**: The module overrides a class from the [Pantheon Apache Solr module](https://pantheon.io/docs/solr-drupal-7) that is responsible for a correct connection to the Pantheon's Apache Solr service. As a result a Solr connection is lost.
+
+**Solution**: Instead of patching the module, you can fix the issue with a custom module:
+
+1. Define a new class that inherits from the `PantheonApachesolrSearchApiSolrService` and contains logic from the `SearchApiSolrDateSortSolrService` (or vice a versa).
```

Comment:
Is defining a new class something done once you have a new custom module, or is it part of the creation of it? I'm not a Drupal person, so I don't know how much of this knowledge can be assumed. 

When you say "contains logic", that's a bit too ambiguous for my taste. Are we copying logic directly from the module in question? Some of it, all of it? Can we provide specific examples in the form of files or excerpts of code?